flake-modules/dev/init-script/default.nix

```diff
@@ -0,0 +1,13 @@
+{
+  pkgs,
+}:
+pkgs.rustPlatform.buildRustPackage {
+  pname = "nixvim-init";
+  version = "nightly";
+
+  src = ./.;
```

Comment:
Generally, `src = ./.` is a bad idea because `default.nix` itself will also be considered a "source" of the derivation.

This means if `default.nix` is modified, the derivation needs to be rebuilt. Even if no source _actually_ changed and the change to the nix file wouldn't actually _need_ a rebuild.

Usually, you'd do something like `src = ./src`. I think you could also use a fileset or filtered source to filter out nix files, but those options seem a little overkill.

That said, this is a minor issue and we're already doing this in most of our other derivations, so it shouldn't block the PR if you'd rather keep it as-is.

Reply:
I still prefer having sources and nix files separate, (I don't think it has any downsides?), but I won't press on the issue.